app/models/topic.rb

```diff
@@ -265,6 +265,19 @@ def from_email_address
     end
   end
 
+  def check_attachments
+    attachments = false
+
+    self.posts.each do |post|
```

Comment:
Looking at the `Topic` model, it seems that an attachment may also be on the `Topic` itself (see [this](https://github.com/firdausmaarof/helpy/blob/2be47ccf3b947eb576e276a5d8251b018bc732f7/app/models/topic.rb#L42)), as well as its associated posts.

Reply:
I think this is true.  I can't remember how it works exactly though.  Good test cases would be 

- Create a new ticket with an attachment using the webform and verify the logo is shown
- Create a new ticket with an attachment by email and verify the logo is shown
- Create a new ticket without an attachment and reply to it with an attachment and verify the logo is shown

These tests already exist, so you can probably just re-use modify the code.